torch/distributed/rpc.py

```diff
@@ -5,7 +5,7 @@
 from . import ProcessGroupAgent
 from . import WorkerId
 from .internal_rpc_utils import serialize, PythonUDF
-from .rpc_backend_handler import is_backend_registered, registered_init_rpc
+from .rpc_init_handler_registry import is_backend_registered, rpc_init
```

Comment:
Does it make sense if we call this `rpc_backend_registery` and `init_backend`? Because we already have a `init_rpc` API in `rpc.py`.

Reply:
Updating